src/content/Animated_Form_webpage.mdx

```diff
@@ -0,0 +1,7 @@
+---
+Title: Animated Login
+Excerpt: A beginner friendly JS animated login/register Ui
+Tags: JavaScript
+FeaturedImg: '../images/BajraYeJoon/page.gif'
```

Comment:
The gif file is missing under `/images/<your-username>` directory. You need to commit it as well.